### ISSUE TITLE
Documentation lacks the part on describing Keycloak integration

### ISSUE THREAD

**Issue by drook** [Apicurio/apicurio-studio#1983]
[Documentation article](https://www.apicur.io/studio/docs/setting-up-keycloak-for-use-with-apicurio) lacks the part where the Keycloak integration is described for on-premises installations.
The json template mentioned isn't well parsed by the KC 13.x, and the mentioned KC 7.x contains like gazillion CVEs.

**Comment by EricWittmann** [comment]
Yeah that article is pretty outdated at this point.  Right now https://studio.apicur.io is protected by KC 16 - so it's not the case the KC 7 is actually required.  That was just the version available at the time of writing the article.
